package.json

```diff
@@ -15,7 +15,7 @@
     "storybook": "start-storybook -p 6006 -s ./",
     "test": "npm run --silent skip-taskcluster || (npm run test-addon && npm run test-support-library)",
     "test-addon": "npm run --silent skip-taskcluster || (mocha --require  \"./tests/hooks.js\" \"./tests/core-addon/unit/*.js\")",
-    "test-integration": "npm run --silent skip-taskcluster || (npm run build && mocha --timeout 30000 \"./tests/core-addon/integration/*.js\")",
+    "test-integration": "npm run --silent skip-taskcluster || (npm run build && mocha --unhandled-rejections=strict --timeout 30000 \"./tests/core-addon/integration/*.js\")",
```

Comment:
Ah, nice. If I understand this correctly, tests should fail earlier with this flag in case of problems, without being falsely marked as positive. Is that correct? 

Would you kindly file an issue for adding this mode to the other test commands as well?

Reply:
Sure! Yes essentially it just errors out if any promises reject and it's never handled by anything, I noticed this when I forgot to do an `await` but the tests seemed to pass anyway (it was a legit error in the test),

Reply:
issue #317